core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,105 @@
+<div class="skill-selector-parent skill-container oppia-skill-container protractor-test-skill-container">
+  <div class="list-group oppia-list-group">
+    <mat-card layout="row" class="list-view-item protractor-test-skill-select-header">
+      Search By Skill
+      <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" ng-model="skillFilterText" autofocus>
+    </mat-card>
+    <div class="skill-list-box">
+      <mat-radio-group ng-model="selectedSkill" ng-change="setSelectedSkillId()">
+        <mat-card *ngFor="(topicName, subtopics) in categorizedSkills" class="list-view-item" *ngIf="checkIfTopicIsEmpty(topicName)">
+          <mat-card layout="row" class="list-view-item">
+            <h2><label class="oppia-topic-name-text">{{ topicName }}</label></h2>
+          </mat-card>
+          <div *ngIf="!checkIfEmpty(skills)" *ngFor="(subtopicName, skills) in subtopics" class="list-view-item ">
+            <mat-card class="list-view-item">
+              <label class="oppia-subtopic-name-text">{{ subtopicName }}</label>
+              <md-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="skill in skills | filter:skillFilterText" ng-value="skill.id" aria-label="{{ skill.description }}">
```

Comment:
I think this should change to mat-radio-button

Reply:
Done